### ISSUE TITLE
Jira & Confluence deployments need the Shared Home volume turned on

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/zarf-package-software-factory#299]
If they don't have the Shared Home volume, they lose data on reboot ("attachments, backups, and other content"). The trick here is gonna be that the Shared Home volume is required to be of type RWX and that isn't a type that a lot of storageclasses support.

**Comment by RothAndrew** [comment]
That's the plan for the users who are on EKS for sure, will need to figure out what to do for other use cases.

Lower priority though, since our first adopter is using EKS

**Comment by jeff-mccoy** [comment]
Google and Microsoft have RWX options--interestingly Azure HA NFS looked to be the simplest of all the cloud options. I was looking into this for HA registry things.

**Comment by RothAndrew** [comment]
Blocked by #401
